scripts/superdesk-search/services/TagService.js

```diff
@@ -23,6 +23,13 @@ export function TagService($location, desks, userList, metadata, search) {
 
     var cvs = search.cvs;
 
+    function tag(label, value) {
+        return {
+            label: label,
+            value: value || label
+        };
+    }
+
     function initSelectedParameters (parameters) {
         tags.selectedParameters = [];
```

Comment:
If the `tag` function is only used in the scope of this function, should it not be here:
```js
let tag = (l, v) => ({label: l, value: v||l});
```
I feel like the ES6 police :-)